test_rest/mocks/v2/snapshot/locale/us/markets/stocks/tickers/AAPL.json

```diff
@@ -47,6 +47,7 @@
         "ticker": "AAPL",
         "todaysChange": -3.65,
         "todaysChangePerc": -2.231,
-        "updated": 1651251948294080343
+        "updated": 1651251948294080343,
+        "fair_market_value": 160.315
```

Comment:
shouldn't this be "fmv", not "fair_market_value"

Reply:
yep, good catch